config/samples/example-with-configmap.yaml

```diff
@@ -19,25 +19,24 @@ data:
       - model_id: "llama3.2:1b"
         provider_id: ollama
         model_type: llm
+        provider_model_id: llama3.2:1b
```

Comment:
Is this needed? Since above is already a `model_id` and a `provider_id` ?

Reply:
not necessary, i can remove it